### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 6024723: character maps to <undefined>

### ISSUE THREAD

**Issue by joseespinadote** [khornberg/octokit.py#119]
After fresh install on Windows using conda, following the example in the docs, I did:

`from octokit import Octokit`

and then I got:

```
>>> from octokit import Octokit
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\josee\Documents\mag\tesis\dev\env_github_app\lib\site-packages\octokit\__init__.py", line 6, in <module>
    from octokit_routes import specifications
  File "C:\Users\josee\Documents\mag\tesis\dev\env_github_app\lib\site-packages\octokit_routes\__init__.py", line 31, in <module>
    specifications = {spec.replace(".json", ""): _load_spec(spec) for spec in specs}
  File "C:\Users\josee\Documents\mag\tesis\dev\env_github_app\lib\site-packages\octokit_routes\__init__.py", line 31, in <dictcomp>
    specifications = {spec.replace(".json", ""): _load_spec(spec) for spec in specs}
  File "C:\Users\josee\Documents\mag\tesis\dev\env_github_app\lib\site-packages\octokit_routes\__init__.py", line 21, in _load_spec
    return _load_json(os.path.join(__location__, "routes", spec_name))
  File "C:\Users\josee\Documents\mag\tesis\dev\env_github_app\lib\site-packages\octokit_routes\__init__.py", line 9, in _load_json
    return json.load(f)
  File "C:\devel\conda\lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "C:\devel\conda\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 6024723: character maps to <undefined>
```

I did a quick fix replacing:

`with open(path, "r") as f:`

with:

`with open(path, "r", encoding="utf8") as f:`

at line 9 in ...\site-packages\octokit_routes\__init__.py inside the virtual enviroment

Thanks for this great library

Bests, José.-

**Comment by khornberg** [comment]
Thank you for reporting this

I think this is solved with 0.15.0

**Comment by joseespinadote** [comment]
yeah, it's fixed now. thanks @khornberg
